drive/google/oauth.go

```diff
@@ -0,0 +1,99 @@
+package google
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+
+	"github.com/asjoyner/shade"
+	"github.com/asjoyner/shade/drive"
+
+	gdrive "google.golang.org/api/drive/v3"
+
+	"golang.org/x/oauth2"
+)
+
+const (
+	clientID     = "1060319012426-gb2f7picjh3dfsoni203rkmsjliqok1f.apps.googleusercontent.com"
+	clientSecret = "jfcCV2384sXBlqdaes26bAA5"
+	authURL      = "https://accounts.google.com/o/oauth2/auth"
+	tokenURL     = "https://accounts.google.com/o/oauth2/token"
+	redirectURI  = "https://localhost"
```

Comment:
If I'm reading all the comments right, this is using the "get a verification, paste it back to the client" flow.  Do you still need to specify a redirectURI / redirectURL?  I have a vague feeling you might have to, to match up with what's granted in the Cloud API Console, but if you can do with out it here, it might avoid confusing someone (like me, in a few months) into thinking that you're using the OAuth flow which makes an HTTP redirect back to a daemon listening on localhost.